mcs/class/corlib/System.IO/DriveInfo.cs

```diff
@@ -169,14 +169,39 @@ public override string ToString ()
 		}
 
 		[MethodImplAttribute (MethodImplOptions.InternalCall)]
-		extern static bool GetDiskFreeSpaceInternal (string pathName, out ulong freeBytesAvail,
+		unsafe extern internal static bool GetDiskFreeSpaceInternal (char *pathName, int pathName_length, out ulong freeBytesAvail,
```

Comment:
There is no need for that to be `internal`.

Reply:
Because the default is the more restrictive private? Agreed.

Reply:
I did another pass on the access and I think they are all correct -- no change  in access to existing signatures, and new signatures are all defaulted to private.

Also, if existing icall had not "Internal" in its name, I added that, when changing the signature from string to const char*.
But no name/access change to existing signatures.

I'm a little nervous about the resulting name overloading and icall-table.c but suspect it is ok.
i.e. where before we might have had one `[icall]FooInternal(string)`, just one `FooInternal` we now have two `[icall]FooInternal(const char*)` and `[not-icall]FooInternal(string)`.

If that is a problem, obvious two suggestions are record signature in icall-def.h as we sometimes do, or further rename like to `FooNative`.